src/report.rs

```diff
@@ -21,25 +21,15 @@ pub trait ReportCategory: Serialize + PartialEq {}
 
 /// Type of the report
 #[derive(Debug, Serialize, PartialEq)]
-pub(crate) enum TransitModelReportCategory {
-    // transfers types
-    TransferIntraIgnored,
-    TransferInterIgnored,
-    TransferOnNonExistentStop,
-    TransferOnUnreferencedStop,
-    TransferAlreadyDeclared,
-    // apply rules
-    ObjectNotFound,
-    InvalidFile,
-    UnknownPropertyName,
-    UnknownPropertyValue,
-    MultipleValue,
-    OldPropertyValueDoesNotMatch,
-    GeometryNotValid,
-    NonConvertibleString,
+pub(crate) enum TransferReportCategory {
```

Comment:
Careful, I renamed `TransitModelReportCategory` into `TransfertReportCategory` and remove the prefix `Transfert` on all the variants of the `enum`.